src/components/shared/ColonyAvatar/ColonyAvatar.tsx

```diff
@@ -36,21 +24,17 @@ const ColonyAvatar = ({
   colonyAddress,
   avatarURL,
   className,
-  colony: {
-    // @ts-ignore
-    name,
-    // @ts-ignore
-    profile,
-  },
+  colony,
   notSet,
   size,
   showLink,
-  prefferThumbnail = true,
+  preferThumbnail = true,
 }: Props) => {
-  const imageString = prefferThumbnail ? profile?.thumbnail : profile?.avatar;
+  const { profile, name } = colony || {};
+  const imageString = preferThumbnail ? profile?.thumbnail : profile?.avatar;
```

Comment:
I'm ok with this addition, especially since we're going to have to use this component in places where we'll want to showcase the full size image rather than the thumbnail _(eg: @chinins 's user profile)_

Just please make sure this pattern makes it's way to the other two components that deal with avatars: user and token.

Note that @Julianahlk is currently working on the user avatar part, so make sure to sync up on this, so that this gets into her changes as well.